src/fetchData.js

```diff
@@ -0,0 +1,38 @@
+export const setData = (location) => {
+    if (location) {
+        fetch(`https://api.openweathermap.org/data/2.5/weather?q=${location}&units=metric&appid=90bda63e7c2bc25d4b3159b92c0ff7a8`).then(function(response){
+            response.json().then(function(data) {
+                console.log(data);
+                displayWeather(data);
+            });
+        }).catch(function(error) {
+            console.log('Fetch Error:', error);
+            });
+    };
+};
+
+export const displayWeather = (data) => {
+    const name = document.querySelector('.dataTitle');
+    const temp = document.querySelector('.tempe');
+    const humText = document.querySelector('.hum');
+    const humidity = document.querySelector('.humValue');
+    const pressText = document.querySelector('.pres');
+    const press = document.querySelector('.presValue');
+    const windText = document.querySelector('.wind');
+    const wind = document.querySelector('.windValue');
+    const contText = document.querySelector('.cont');
+    const cont = document.querySelector('.contValue');
+    const img = document.getElementById('img')
```

Comment:
- [OPTIONAL] It'll be good if the background color changes also based on the current weather https://www.loom.com/share/6e576b1ca9194323b070692bded9df7c

Reply:
ok ma'am, thanks..